packages/authorization-process/src/services/authorizationService.ts

```diff
@@ -840,6 +840,31 @@ export function authorizationServiceBuilder(
         )
       );
     },
+    async getProducerKeychainUsers({
+      producerKeychainId,
+      organizationId,
+      logger,
+    }: {
+      producerKeychainId: ProducerKeychainId;
+      organizationId: TenantId;
+      logger: Logger;
+    }): Promise<{ users: UserId[]; showUsers: boolean }> {
+      logger.info(
+        `Retrieving users of producer keychain ${producerKeychainId}`
+      );
+      const producerKeychain = await retrieveProducerKeychain(
+        producerKeychainId,
+        readModelService
+      );
+      assertOrganizationIsProducerKeychainProducer(
+        organizationId,
+        producerKeychain.data
+      );
+      return {
+        users: producerKeychain.data.users,
+        showUsers: true,
+      };
```

Comment:
I think you don't need to return the `showUsers` flag here
```suggestion
      return producerKeychain.data.users;
```

Reply:
Let's see what the others have to say